preview-maker.js

```diff
@@ -129,44 +129,34 @@ const addWallpaper = async (elements, wallpaper) => {
   return result;
 };
 
-function calculateAccentColor(colors) {
-  let colorsQuantity = new Map();
-  let max = 0;
-  let accentHue;
-
+const accentColor = (colors, grey) => {
+  const quantity = new Map();
+  let accentQuantity = 0;
+  let accent;
   for (const color of colors) {
-    colorsQuantity.set(color, (colorsQuantity.get(color) ?? 0) + 1);
-
-    if (colorsQuantity.get(color) > max) {
-      accentHue = color;
-      max = colorsQuantity.get(color);
+    quantity.set(color, (quantity.get(color) ?? 0) + 1);
+    if (quantity.get(color) > accentQuantity) {
+      accent = {
+        background: { hue: color, saturation: 1, lightness: 0.9 },
+        shadow: { hue: color, saturation: 1, lightness: 0.02 },
+      };
+      accentQuantity = quantity.get(color);
     }
   }
-  const colorsHue = colors.filter((color) => colorsQuantity.get(color) == max);
-  colorsHue.sort((a, b) => a - b);
-
-  if (colorsHue.length > 1) {
-    let minDifference = 360;
-    for (let i = 0; i < colorsHue.length - 1; i++) {
-      const outerHue = colorsHue[i];
-      const innerHue = colorsHue[i + 1];
-      if (Math.abs(outerHue - innerHue) < minDifference) {
-        minDifference = Math.abs(outerHue - innerHue);
-        accentHue = (outerHue + innerHue) / 2;
-      }
-    }
+  if (grey / 6 > colors.length) {
```

Comment:
This is a strange way of saying `if the amount of gray variable values is greater than the amount of all other values...`